precompiles/xvm/src/tests.rs

```diff
@@ -68,6 +70,7 @@ fn correct_arguments_works() {
                     .write(0x1Fu8)
                     .write(Bytes(b"".to_vec()))
                     .write(Bytes(b"".to_vec()))
+                    .write(U256::one())
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `write()` function is called with `U256::one()`, but it doesn't seem to be used. Please review and remove this line if it's not needed.
```diff
-                     .write(U256::one())
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->